updatehub/states.go

```diff
@@ -451,6 +455,35 @@ func NewDownloadingState(updateMetadata *metadata.UpdateMetadata, pti ProgressTr
 	return state
 }
 
+// FIXME: add this
```

Comment:
This is still valid? if yes please make it clearer as I did not get what is missing.